net/FlatBuffers/ByteBuffer.cs

```diff
@@ -414,31 +388,26 @@ private void AssertOffsetAndLength(int offset, int length)
 #endif
         }
 
-#if UNSAFE_BYTEBUFFER
+#if ENABLE_SPAN_T
 
-        public unsafe void PutSbyte(int offset, sbyte value)
+        public void PutSbyte(int offset, sbyte value)
         {
             AssertOffsetAndLength(offset, sizeof(sbyte));
-            _buffer.Buffer[offset] = (byte)value;
+            _buffer.GetSpan(offset)[0] = (byte)value;
         }
 
-        public unsafe void PutByte(int offset, byte value)
+        public void PutByte(int offset, byte value)
         {
             AssertOffsetAndLength(offset, sizeof(byte));
-            _buffer.Buffer[offset] = value;
+            _buffer.GetSpan(offset)[0] = value;
         }
```

Comment:
These PutByte and PutSbyte methods are going to be relatively expensive (compared to the cost of storing a byte).  Hopefully they're not used on hot paths.

Reply:
Agreed.